packages/terra-clinical-no-data-view/tests/nightwatch/no-data-view-spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-clinical-no-data-view', done);
+  },
+
+  'Displays a default error view': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/no-data-view-tests/default`)
+      .assert.elementPresent('.terraClinical-NoDataView');
+  },
+  'Displays an error view with text for name and description': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/no-data-view-tests/text`)
+      .assert.containsText('.terraClinical-NoDataView .terraClinical-NoDataView-heading', 'test heading')
+      .assert.containsText('.terraClinical-NoDataView .terraClinical-NoDataView-subtext', 'test subtext')
+      .assert.elementPresent('.terraClinical-NoDataView');
+  },
+  'Displays an error with a button': (browser) => {
```

Comment:
Is this the test description you intended to use? Same question for test on line 19.